docs/source/contributing.generator.rst

```diff
@@ -189,13 +206,18 @@ The next step is to instantiate the class. Let's try with that ``meta/llama-2-70
 .. code-block:: bash
 
     >>> g = garak.generators.replicate.ReplicateGenerator("meta/llama-2-70b-chat")
-    🦜 loading generator: Replicate: meta/llama-2-70b-chat
     Traceback (most recent call last):
     File "<stdin>", line 1, in <module>
-    File "/home/lderczynski/dev/garak/garak/generators/replicate.py", line 42, in __init__
-        raise ValueError(
-    ValueError: 🛑 Put the Replicate API token in the REPLICATE_API_TOKEN environment variable (this was empty)
-                    e.g.: export REPLICATE_API_TOKEN="r8-123XXXXXXXXXXXX"
+    File "/home/lderczynski/dev/garak/garak/generators/replicate.py", line 44, in __init__
+        super().__init__(name, generations=generations, config_root=config_root)
+    File "/home/lderczynski/dev/garak/garak/generators/base.py", line 43, in __init__
+        self._load_config(config_root)
+    File "/home/lderczynski/dev/garak/garak/configurable.py", line 60, in _load_config
+        self._validate_env_var()
+    File "/home/lderczynski/dev/garak/garak/configurable.py", line 116, in _validate_env_var
+        raise APIKeyMissingError(
+    garak.exception.APIKeyMissingError: 🛑 Put the Replicate API key in the REPLICATE_API_TOKEN environment variable (this was empty)
+                            e.g.: export REPLICATE_API_TOKEN="XXXXXXX"
```

Comment:
This stack trace is a great example of how `Configurable` handles the `ENV_VAR` load for the generator without the author having to do it.